content/90.back-matter.md

```diff
@@ -1,5 +1,5 @@
 {% if format is defined and format != "tex"-%}
```

Comment:
```suggestion
{% if format is not defined or format is defined and format != "tex"-%}
```

I broke this when working on the LaTeX export.  It should be modified like this.